Jenkinsfile

```diff
@@ -1,22 +1,56 @@
 def nodes = nodesByLabel('HIL')
-def boards = []
-def tests = []
-def nodeMap = [:]
 
-def triggers = []
-
-if ("${env.BRANCH_NAME}" == 'nightly') {
-    // build master with latest RIOT daily at 1:00 AM
-    triggers << parameterizedCron('0 1 * * * % HIL_RIOT_VERSION=master')
+pipeline {
+    agent { label 'master' }
+    options {
+      timeout(time: 1, unit: 'HOURS')
+      parallelsAlwaysFailFast()
+    }
+    stages {
+        stage('setup') {
+            steps {
+                stepClone()
+                stash name: 'sources'
+                script {
+                    for (i = 0; i < nodes.size(); i++) {
+                        echo "${nodes[i]}"
```

Comment:
is this `script` block a left-over from debugging, or do you want to keep it for informational purposes?

Reply:
Debugging, thanks!